src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ManagedACTAccessRequirementWidgetViewImpl.java

```diff
@@ -419,8 +419,4 @@ public void setSubjectsDefinedByAnnotations(
     subjectsDefinedByAnnotationsUI.setVisible(v);
     subjectsDefinedInAccessRequirementUI.setVisible(!v);
   }
-
-  public void hideRequestAccessModal() {
-    requestDataAccessWidget.clear();
```

Comment:
Clearing widgets did not unmount the react component, which is what we need to do in this case (since it will display the modal if it is mounted and configured to show itself as a modal).  So instead wrap the ReactComponent in a GWT SimplePanel (div that allows a single child), and add the ReactComponent when it should be shown (and remove when it should be hidden)

Reply:
That works